design-system/materials/internals/definition.yaml

```diff
@@ -99,7 +99,7 @@ choiceGroupsByTheme:
       label: Font families
       prefix: font-family
       choices:
-        font-family: "'Open Sans', sans-serif"
+        font-family: "'Inter', system-ui"
```

Comment:
The changes in this file consist in overwriting the default theme values with the ones we had in the test one and remove the latter so we just keep the `default` one with the new styles values.

We introduced many new design tokens for the transition from one look and feel to the other. We're keeping those ones by now since we're not sure how much granularity we want to have with the tokens. That would be an upcoming project.